docs/pages/database-access/guides/elastic.mdx

```diff
@@ -0,0 +1,248 @@
+```yaml
+db_service:
+  enabled: true
+  resources:
+  - labels:
+      "*": "*"
+  databases:
+  - name: myelastic
+    protocol: elasticsearch
+    uri: {ELASTICSEARCH_HOST}:9200
+    static_labels:
+      env: dev
+```
+
+Adjust for your environment, then start or restart Teleport.
+
+</TabItem>
+</Tabs>
+
+<Admonition type="note">
+
+The `--auth-server` flag must point to the Teleport cluster's Proxy Service
+endpoint because the Database Service always connects back to the cluster over a
+reverse tunnel.
+
+</Admonition>
+
+<Admonition type="tip">
+  You can start the Database Service using a configuration file instead of CLI flags.
+  See the [YAML reference](../reference/configuration.mdx) for details.
+</Admonition>
+
+## Step 2/5. Create or modify a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+Alternately, you can edit an existing user with database access.
+
+1. As a Teleport user with the `editor` role, import a user's configuration to a local `yaml` file:
+
+   ```code
+   $ tctl get user/{teleport-user} > user.yaml
+
+   ```
+
+2. Modify `spec.traits.db_names` and `spec.traits.db_users` with the name(s) and database user(s) this user should have access to, or use `*` to provide access to all databases/users:
+
+    ```yaml
+    ...
+    spec:
+      ...
+      traits:
+      ...
+      db_names:
+      - '*'
+      db_users:
+      - '*'
+      ...
+    ```
+
+3. Apply the changes to the Teleport user:
+
+   ```code
+   $ tctl create -f user.yaml
+   user "teleport-user" has been updated
+   ```
+
+## Step 3/5. Create Role Mapping
+
+Define a role mapping in Elasticsearch to assign your Teleport user or role to an Elasticsearch role.
+The example below maps the Teleport user `alice` to the `superuser` role in Elasticsearch, which
+should **not** be done in production:
+
+```code
+$ curl -u elastic:{ELASTICSEARCH_USER_PASSWORD} -X POST "https://{ELASTICSEARCH_HOST}:9200/_security/role_mapping/mapping1?pretty" -H 'Content-Type: application/json' -d'
+{
+  "roles": [ "superuser"],
+  "enabled": true,
+  "rules": {
+    "field" : { "username" : "alice" }
+  },
+  "metadata" : {
+    "version" : 1
+  }
+}
+'
+```
+
+## Step 4/5. Set up mutual TLS
+
+(!docs/pages/includes/database-access/tctl-auth-sign.mdx!)
+
+We will show you how to use `tctl auth sign` below.
+
+To securely access your Elasticsearch database, sign the certificate for the hostname Teleport will
+connect to:
+
+For example, if your Elasticsearch server is accessible at `elastic.example.com`,
+run:
+
+```code
+$ tctl auth sign --format=elastic --host=elastic.example.com --out=elastic --ttl=2160h
+Database credentials have been written to elastic.key, elastic.crt, elastic.cas.
+```
+
+(!docs/pages/includes/database-access/ttl-note.mdx!)
+
+The command will create three files:
+
+- `elastic.cas` with Teleport's certificate authority
+- `elastic.key` with a generated private key
+- `elastic.crt` with a generated user certificate
```

Comment:
Is this a user certificate or a host certificate?

Reply:
Updated to say host certificate.